src/SFA.DAS.ApprenticeCommitments/DTOs/ApprenticeshipDtoMapping.cs

```diff
@@ -43,7 +46,42 @@ public static class ApprenticeshipDtoMapping
                 ApprovedOn = latest.CommitmentsApprovedOn,
                 ChangeOfCircumstanceNotifications = apprenticeship.ChangeOfCircumstanceNotifications,
                 StoppedReceivedOn = latest.StoppedReceivedOn,
+                Revisions = GetRevisions(apprenticeship.Revisions),
```

Comment:
I think you have added this to the mapper which gets the latest revision. I think this wrong you need to add this functionality to the mapper which gets the latest Confirmed apprenticeship revision

Reply:
yes it was added to the mapper intentionally - the apprenticeship model deals with EF models, the mapper deals with the DTOs.  It looks neater when the model doesnt reference the DTOs.  The apprenticeship model has a LatestRevision property which I think is what you are referring to. The apprenticeship model also has a list of all associated Revision entries under property 'Revisions' which is what im using..